app/jobs/create_talroo_job_posting_job.rb

```diff
@@ -0,0 +1,82 @@
+class CreateTalrooJobPostingJob < ApplicationJob
+  queue_as :job_posting
+
+  SOURCE = ENUMS::JOB_SOURCES::TALROO
+
+  def perform(xml, tags)
+    setup_tags(tags)
+
+    fragment = Nokogiri::XML(xml)
+    return unless fragment.children.present?
+
+    job = extract_job_params(fragment)
+    JobPosting.create(job) unless job.nil?
```

Comment:
We may want to call `create!` here... though it could be noisy if there are lots of errors on save. I'm just worried that we may swallow records that don't save due to a smaller validation error or something. Thoughts?